lib/index.js

```diff
@@ -280,7 +280,7 @@ Metalsmith.prototype.readFile = unyield(function*(file){
 
     if (frontmatter && utf8(buffer)) {
       try {
-        parsed = matter(buffer.toString());
+        parsed = matter(buffer.toString().replace(/\r\n/g, '\n'));
```

Comment:
gray-matter fails on this line: https://github.com/jonschlinkert/gray-matter/blob/master/index.js#L98. I originally changed this line to account for \r\n which fixed the problem but the usage as described [here](https://www.npmjs.com/package/gray-matter) implies that \r\n is not supported.